libs/deps/literals/price/ar.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
Should all of those really live in-code? The franklin/milo way would be to store those in sharepoint in a excel sheet and retrieve them from there

Reply:
@mokimo they will be generated from an endpoint, they are not author-able.
Having said that, we might imagine generating and saving them in excel in the future, not sure if it is better.

Reply:
@mokimo @yesil  I am not sure which will be the advantage of putting them in a excel sheet. It could be that it will just increase the complexity.